source/documentation/runbooks/010-disaster-recovery-plan.html.md.erb

```diff
@@ -0,0 +1,129 @@
+| Dependency poisoning attacks                                                                                       | High     | Low        | - Using Dependabot<br>- Scanning on pipeline - NPM audit<br>- Container scanning<br>Dependency assessment criteria<br>Guardduty monitors VPC flow logs for unusual activity |
+| Accidental misconfiguration of code leading to data loss                                                           | High     | Low        | - Manually fix in the DB where possible<br>- DB snapshot	restore |
+| Entra ID disruption (loss of authentication function)                                                              | High     | Low        | - Wait |
+
+## Backup Procedures
+
+### Django application
+- **Application code**:
+   - All application code is pushed to the [public GitHub repository].
+
+### Databases
+- **PostgreSQL database in AWS RDS (Find MoJ data, DataHub)**:
+    - RDS automated daily snapshots are enabled by default in CP.
+    - Ensure point-in-time recovery is active.
+    - `TODO` Schedule regular manual snapshots and store them in a different AWS region.
+
+    >**Note** manual snapshots set up this way must also be deleted, as there is an account-level snapshot quota.
+- **OpenSearch (DataHub)**:
+    - OpenSearch index snapshots are [handled by Cloud Platform as part of OpenSearch deployment]
+
+## Recovery Procedures
+
+### Django application
+<placeholder>
+
+### Microsoft Entra ID integration
+<placeholder>
+
+### Database recovery
+- **Restore PostgreSQL database from RDS snapshots (Find MoJ data, DataHub)**:
+   - If needed, restore the database to a specific point in time [following CP's RDS snapshot restore guidance].
+- **Restore OpenSearch index from snapshots in S3 (DataHub)**:
+   - Recover OpenSearch indices from the latest snapshots stored in S3 ([AWS instructions])
+   - Verify search functionality is restored in the application.
+
+### Issues at Cloud Platform level
+- **Redeploy environments**:
+   - Once Cloud Platform have confirmed that their platform is available, re-deploy affected environment applications
+   - Trigger GitHub Actions deployments either through manual re-trigger of the latest workflow run, or through merging a PR to `main` and running the relevant deploy pipelines.
+
+## Communication Plan
+- **Internal Communication**:
+  - Notify the Data Catalogue team immediately when a disaster is identified via either of [#data-catalogue] or [#data-catalogue-alerts] Slack channels.
+  - Provide regular updates to stakeholders throughout the recovery process.
+- **External Communication**:
+  - Inform users about any relevant service disruptions via the [#ask-data-catalogue] Slack channel and the [Ask Data Catalogue] Teams channel. Provide updates until full restoration.
+
+## Post-Disaster Review
+- **Incident Analysis**:
+   - Conduct a post-mortem to identify the root cause of the disaster.
```

Comment:
Small wording change suggestion - if it's serious enough to call a disaster there'll be more than one thing to discuss. (Also some people find "post-mortem" a bit grim!)

```suggestion
   - Conduct an incident review to identify contributing factors to the disaster.
```

Do you know if MoJ have a template for this that we can link to?

Reply:
Good suggestion. Accepted. 

I'm not sure of a shared / template process for post-incident review. For the incident response I do quite like [CP's current process](https://runbooks.cloud-platform.service.justice.gov.uk/incident-process.html)